### ISSUE TITLE
Questions about DB states.

### ISSUE THREAD

**Issue by xiami2019** [bepoetree/MTTOD#5]
Hi, nice work!
A little question here is that it seems like you doesn't use DB state when training e2e task.
`resp_outputs = self.model(attention_mask=attention_mask,
                                      encoder_outputs=encoder_outputs,
                                      lm_labels=resp_labels,
                                      return_dict=False,
                                      decoder_type="resp")`

**Comment by bepoetree** [comment]
Hi!

The "resp_labels" consists of [db_state, system_action, response]. In the inference phase, the decoder has db_state as initial tokens.

For more details, please refer to reader.py (275 line):
`resp = turn["dbpn"] + turn["aspn"] + turn["redx"]`

**Comment by xiami2019** [comment]
Got it, thanks for your reply :)

**Comment by xiami2019** [comment]
Hi, I think it may be a little wrong here. The model should not learn to predict db_state tokens, thus I think it doesn't make sense taking db_state tokens as labels. Maybe it can be used as the inputs of decoder both when training and evaluation.
